### PR TITLE
Misinformation SEO page (Fixes #9957)

### LINKED ISSUE #9957: Misinformation SEO page
## Description

Brief / copy doc: https://docs.google.com/document/d/1-jgNXYJVAcFgzcQrKKJTglG1wfv5mYjcZvPYCS5ZSEg/edit?usp=sharing (Mozilla access only).

---

#### Success Criteria
- [ ] Page is launched and available in Pontoon for L10n.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
http://localhost:8000/en-US/firefox/more/misinformation/

## Issue / Bugzilla link
#9957

## Testing
https://www-demo2.allizom.org/en-US/firefox/more/misinformation/

### CODE REVIEW

**Comment by alexgibson** [comment]
Ready for code review (but still DNM until we get final copy approval)

**Comment by alexgibson** [comment]
This has been approved by stakeholders and is good for review/merge.

**Review by craigcook — CHANGES_REQUESTED** [review]
Just a few tiny nitpicks but no show-stoppers.

I do have some concerns about the page content. In the recent past some people have falsely accused Mozilla and Firefox of attempting to "censor" or "filter" content, given our stance against the spread of misinformation online. Ironically that is itself misinformation, since Firefox does no such thing.

But I could see this page being misinterpreted to support that false narrative, especially if someone only reads the headline and doesn't read the article, and fills in the rest with their mistaken assumptions.

Maybe I'm being overly cautious, but we should run this past the comms team for input and to assess the risk of trolls citing this page in further misinformation campaigns. Maybe we could a some clear, emphatic statement that Firefox in no way filters content and doesn't prevent users from accessing false or misleading information if that's what they want to see.

File: l10n/en/firefox/more/misinformation.ftl
Comment (craigcook) [inline l10n/en/firefox/more/misinformation.ftl:46]: Since this is only shown for English and German, should we omit it from Fluent and hardcode it in the template? Or if we want to have translations handled in Pontoon maybe just add a comment that it's only shown in EN and DE?

File: bedrock/firefox/templates/firefox/more/misinformation.html
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/more/misinformation.html:5]: Good catch, I see no reason why this can't use the macro. Will update!

File: l10n/en/firefox/more/misinformation.ftl
Comment (alexgibson) [reply l10n/en/firefox/more/misinformation.ftl:46]: This is shown the en and de right now since only those locales get Pocket Recommendations. I don't have the German translation yet, so can't hard code it alas. But I guess there's also a possibility that this might include fr at some point in the future. 

I'll add a comment to the string 👍

**Comment by alexgibson** [comment]
Thanks for the review @craigcook, updated.

I'll speak to Anika today re: your concern and update here when I hear back.

**Comment by alexgibson** [comment]
@craigcook I've added an updated page headline / subtitle, but let's hold off merging until we know everything is final. I've asked if we should also change the meta description to reflect the new phrasing also.

**Comment by alexgibson** [comment]
@craigcook ok copy is final, we're good to go here

**Review by craigcook — APPROVED** [review]

**Pull request merged**
